CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## upcoming version
```

Comment:
Will this be a major version bump?

Reply:
I haven't decided on that, yet.

Reply:
I see that the package is published to pub.dev, we should defently do a major version bump when releasing the update as it might break production apps.

It would be great to update the readme to include adding the dependency via pub and not via git 👍🏽

Reply:
Alright other idea:
We label all the bugs and issues that should be in the stable 1.0 release. This means we fix all the platform inconstencies, we make sure all crashes are fixed and then we release a major version update to pub.dev 👍🏽

Reply:
> I see that the package is published to pub.dev, we should defently do a major version bump when releasing the update as it might break production apps.
Interesting, I didn't know about that :) I assume that was done by the MapLibre team.

> It would be great to update the readme to include adding the dependency via pub and not via git 👍🏽
👍

Reply:
> I see that the package is published to pub.dev


In case you didn't see it, yet, I asked and it turns out the pub.dev package was not released by someone from the MapLibre team (see #266)